ansible_builder/main.py

```diff
@@ -49,18 +52,39 @@ def build_command(self):
             self.build_context
         ]
 
+    def run_in_container(self, command, **kwargs):
+        wrapped_command = [self.container_runtime, 'run','--rm']
+
+        wrapped_command.extend(['-v', f"{os.path.abspath(self.build_context)}:/context:Z"])
```

Comment:
If we're hardcoding this here, we can remove `volumes` from the kwargs passed to the invocations below.

Reply:
Yes, did that. I wanted this to be an assumption about running in the container.

One more thing to note - `abspath` make the syntax more verbose

```
docker run --rm -v /Users/alancoding/Documents/repos/ansible-builder/test/data/bc:/context:Z exec-env-pycurl /bin/bash -c python3 /context/introspect.py
```

I don't know that it's necessary. It's clearly not broken, so I don't want to touch it now, just submitting it for consideration. It could be the case that it has `..` in it, so if we got rid of this, maybe we would have to resolve the relative path, which might take a little more legwork.